mysocial/comment/tests.py

```diff
@@ -1,3 +1,78 @@
-from django.test import TestCase
+from fcntl import F_FULLFSYNC
+from rest_framework.test import APITestCase
```

Comment:
Ah, one thing!  I don't have the library for this on my end (I'm using Windows). It doesn't seem to be used elsewhere, so I removed `from fcntl from F_FULLSYNC` on my end to make the test work.

```suggestion
from rest_framework.test import APITestCase
```